.gitignore

```diff
@@ -1,3 +1 @@
-test.md
-node_modules/
-
+node_modules
```

Comment:
- [ ] According to the project requirement you should create a test.md file with the text "This file should be ignored by git" and that file should be ignored by the git.